modules/fx/arrow-fx/src/main/kotlin/arrow/fx/KindConnection.kt

```diff
@@ -1,346 +0,0 @@
-package arrow.fx
```

Comment:
Feels like you're bringing the old `IOConnection` back and no `KindConnection` anymore. Could you give more details on this?

Reply:
That's correct. `KindConnection` is removed from public API and `IOConnection` becomes internal in `arrow.fx`. See other reply about why `KindConnection` was too error-prone.